### PR TITLE
Document readonly classes

### PR DESCRIPTION
**Pull request by Girgias**
RFC: https://wiki.php.net/rfc/readonly_classes

### CODE REVIEW

File: language/oop5/basic.xml [hunk 1]
```diff
@@ -125,6 +125,66 @@ Stack trace:
...
+    <title>Readonly classes</title>
+    <para>
+     As of PHP 8.2.0, a class can be marked with the
+     <literal>readonly</literal> modifier.
+     Marking a class as <literal>readonly</literal> will add the
+     <literal>readonly</literal> modifier to every declared property,
+     and prevent the creation of
+     <link linkend="language.oop5.properties.dynamic-properties">dynamic properties</link>.
+     Moreover, it is impossible to add support for them by using the
+     <code>#[AllowDynamicProperties]</code> attribute. Attempting to do so
```
Comment (cmb69) [inline language/oop5/basic.xml:139]: Maybe refer to the class directly:
```suggestion
     <classname>AllowDynamicProperties</classname> attribute. Attempting to do so
```

File: language/oop5/basic.xml [hunk 2]
```diff
@@ -125,6 +125,66 @@ Stack trace:
...
+     will trigger a compile-time error.
+    </para>
```
Comment (kocsismate) [inline language/oop5/basic.xml:141]: maybe you could mention that readonly classes can extend only readonly classes, and non-readonly classes cannot extend readonly ones either.

File: language/oop5/basic.xml [hunk 3]
```diff
@@ -125,6 +125,66 @@ Stack trace:
...
     </example>
    </warning>
+
+   <sect3 xml:id="language.oop5.basic.class.readonly">
+    <title>Readonly classes</title>
+    <para>
+     As of PHP 8.2.0, a class can be marked with the
+     <literal>readonly</literal> modifier.
+     Marking a class as <literal>readonly</literal> will add the
+     <literal>readonly</literal> modifier to every declared property,
```
Comment (mumumu) [inline language/oop5/basic.xml:135]:
```suggestion
     <link linkend="language.oop5.properties.readonly-properties"><literal>readonly</literal> modifier to every declared property</link>,
```

File: language/oop5/basic.xml (hunk 2 above)
Comment (Girgias) [reply language/oop5/basic.xml:141]: Added a paragraph :)

**Review by cmb69 — APPROVED** [review]
Thank you!

**Pull request merged**